### ISSUE TITLE
NFT preview is sometimes rotated 90 degrees

### ISSUE THREAD

**Issue by moloch10** [asastats/channel#970]
Bug description:

I noticed this while browsing people's collections:

[Example 1](https://www.asastats.com/JHMGC2VUKAFKEYVR5NXFFYNPB5PDDWNLPBCCHSMKUWWBIWJQNRHBWZVCYM)
![image](https://github.com/asastats/channel/assets/97535612/8d76d710-78ea-4b40-842e-ee86e2df2dc2)

[Example 2](https://www.asastats.com/YL442SRCBSML5FQPGIYKWVN4FOUPZ6OMHOXAWJEFBRPBH6BSSJWNY5SJ6M)
![image](https://github.com/asastats/channel/assets/97535612/24a1c3b5-e57a-431a-942c-190299caba38)

**Comment by ipaleka** [comment]
Pretty strange, tnx...

**Comment by moloch10** [comment]
Yeah, I suspect it might have something to do with these specific collections. I see on another rotated AlgoBones on a different wallet than the ones above.

**Comment by ipaleka** [comment]
The issue is in our engine, which doesn't read the JPEG metadata correctly.
